src/App.tsx

```diff
@@ -1,16 +1,27 @@
+import { FC, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
+
+export const App: FC = () => {
+  const [movies, setMovies] = useState<Movie[]>(moviesFromServer);
+
+  const onAdd = (movie: Movie) => {
+    setMovies((prew) => [
+      ...prew,
+      movie,
+    ]);
```

Comment:
Use more descriptive names
```suggestion
    setMovies(prevMovies => [...prevMovies, movie ]);
```